packages/cache-s3/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: Markus Schirp <mbj@schirp-dso.com>
+# Maintainer: Markus Schirp <mbj@schirp-dso.com>
+arch=x86_64
+gitref=515ff24590419e4fdb5aa3a392dcb815801a9a04
+license=BSD-3-Clause
+makedepends="ghc=8.4.3-r0 stack>=1.9.3 zlib-dev"
+pkgdesc='Utility to cache stack state on S3'
+pkgname=cache-s3
+pkgrel=0
+pkgver=0.1.5
+dirname="cache-s3-$gitref"
+filename="$gitref.tar.gz"
+source="https://github.com/mbj/cache-s3/archive/$filename"
+url=https://www.haskellstack.org
+
+build() {
+  cd "$srcdir/$dirname"
+
+  stack build                 \
```

Comment:
Is the`build` argument positional and does `stack` respect the `--` convention for bash commands? If yes to both, I would use `stack $args -- build` for this scheme.

Reply:
The flags I pass are specific to the `build` subcommand. Moving the flags to the suggested position would fail the stack CLI parser.